atomic_reactor/plugins/pre_tag_from_config.py

```diff
@@ -108,7 +112,8 @@ def get_component_name(self):
 
     def run(self):
         self.lookup_labels()
-        return self.parse_and_add_tags()
+        self.parse_and_add_tags()
+        self.add_registry_to_images()
 
     def lookup_labels(self):
         if self.workflow.build_result.is_image_available():
```

Comment:
I didn't notice this before - this should probably be removed, now that the plugin runs in prebuild it shouldn't be checking the build result.

And please replace the df_parser call below with the BuildDir equivalent (dockerfile_with_parent_env)

Reply:
updated